catalog/tests/test_models.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+
+from catalog.models import Author
+
+
+class AuthorModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
method setUpTestData vs setUp có gì khác biệt nhỉ ?

Reply:
Hai method này thì em không rõ lắm, em tra thì được thông tin:
Trong testing với Django, setUpTestData() và setUp() đều được sử dụng để thiết lập các dữ liệu mẫu để sử dụng trong quá trình testing.Tuy nhiên, có một số khác biệt giữa hai phương thức này:

- setUpTestData() chỉ được chạy một lần duy nhất khi các bộ test được chạy, ngay trước khi các bộ test đầu tiên được chạy. Nó được sử dụng để thiết lập dữ liệu mẫu lớn hoặc dữ liệu cơ sở dữ liệu, giúp giảm thiểu thời gian thiết lập dữ liệu trong các bộ test riêng lẻ.

- setUp() được gọi trước mỗi bộ test và được sử dụng để thiết lập các đối tượng, dữ liệu mẫu hoặc các biến toàn cục mà các bộ test riêng lẻ cần. Nó giúp đảm bảo rằng các bộ test được thực thi trong một trạng thái độc lập và không phụ thuộc vào các bộ test khác.